package.json

```diff
@@ -55,6 +55,7 @@
   "devDependencies": {
     "@eslint/js": "^9.9.0",
     "@statelyai/inspect": "^0.4.0",
+    "@total-typescript/ts-reset": "^0.6.1",
```

Comment:
Can you give some specific reasons why this is needed? I read up on the library, but it is still unclear to me exactly what we want to achieve with it in our project

Reply:
Well it makes it possible to get rid of few `//@ts-expect-error` and makes it unnecessary to explicitly type few things ( like `foo as SomeType` ). But overall it's not so necessary. 

Lets remove it?

Reply:
Mmm ok that sounds useful, but maybe adding a whole new dependency for something like that is not the best idea  at this stage of the project, I think we should remove if it doesnt bring significant benefits👍🏻